test/unstable-keyword/module/unstable_module.chpl

```diff
@@ -0,0 +1,18 @@
+//linear algebra
+//compilation error
+use LinearAlgebra;
+
+@unstable "The module 'Linear Algebra' is unstable" var A = Matrix(4,4, eltType=int);
+A = 3;
+writeln(A);
+//Prints:
+  //   3 3 3 3
+  //   3 3 3 3 
+  //   3 3 3 3
```

Comment:
This is incorrect, looks like it outputs 4 rows but you only show 3 here.  I'd recommend updating it